server.js

```diff
@@ -0,0 +1,306 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const qs = require('querystring');
+const adminPrivilege = {
+  brett: 'password',
+};
+
+const server = http.createServer((req, res) => {
+  if (req.method === 'GET') {
+    if (req.url === '/') {
```

Comment:
Instead of having an if statement for each file that you're serving, you should rewrite it so that it can take in any file that the user specifies. This way, as you create new files through your POST method, you won't need to update your code to handle new files.